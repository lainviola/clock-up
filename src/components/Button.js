import styled from 'styled-components'

export const Button = styled.button`
  font-size: 1em;
  line-height: 28px;
  padding: 0.25em 1em;
  border-radius: 3px;
  cursor: pointer;
`
