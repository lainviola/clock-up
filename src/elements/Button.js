import styled from 'styled-components'

export const Button = styled.button`
  color: ${props => (props.primary ? '#ffffff' : props.color)};
  background-color: ${props => (props.primary ? props.color : '#ffffff')};
  font-size: 1em;
  line-height: 28px;
  padding: 0.25em 1em;
  border-radius: 3px;
  border: 2px solid ${props => props.color};
  cursor: pointer;
  &:focus {
    outline: 0;
  }
`
