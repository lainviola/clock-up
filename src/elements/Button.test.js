import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Button } from './Button'

describe('<Button />', () => {
  it('should match to spapshot', () => {
    const wrapper = shallow(<Button />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
