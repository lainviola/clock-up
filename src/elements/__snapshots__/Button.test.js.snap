// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should match to spapshot 1`] = `
<button
  className="Button-kDSBcD cGlyDV"
/>
`;
